src/ops/cli/aws.py

```diff
@@ -9,9 +9,19 @@
 #governing permissions and limitations under the License.
 
 from . import get_output
+from datetime import datetime, timezone
 
 def acess_key(profile):
     return get_output('aws configure get aws_access_key_id --profile %s' % profile)
 
 def secret_key(profile):
     return get_output('aws configure get aws_secret_access_key --profile %s' % profile)
+
+def expiry_dttm(profile):
+    return get_output('aws configure get expiry_dttm --profile %s' % profile)
+
+def expiry_dttm_in_minutes(profile):
```

Comment:
Not familiar with the `dttm` acronym, what does it stand for? (delta time?)
To make things more clear and easy to understand, what do you think about renaming to something like `minutes_until_expire` / `time_until_expire_minutes` / `time_remaining_minutes` etc.

Reply:
I assume `dttm` means datetime, I used the same terminology as in the AWS config. I'll rename to `minutes_until_expire`